src/applications/financial-status-report/constants/deduction-codes/index.js

```diff
@@ -10,210 +7,3 @@ export const deductionCodes = Object.freeze({
   '74': 'Post-9/11 GI Bill debt for tuition',
   '75': 'Post-9/11 GI Bill debt for tuition (school liable)',
 });
-
-export const renderAdditionalInfo = deductionCode => {
```

Comment:
Oops I just realized that this team generally split CDP and FSR changes into separate PRs. I did find that both `renderAdditionalInfo` and `renderWhyMightIHaveThisDebt` are unused in FSR so I removed them as well but let me know if I should revert this or do it in a separate PR.

Reply:
This is minor enough it should be fine! 🤙